### ISSUE TITLE
Is this intentional?

### ISSUE THREAD

**Issue by salemalem** [chibat/chibat.github.io#1]
![image](https://user-images.githubusercontent.com/42411937/118401409-69fd9a00-b687-11eb-9668-0113ef435c39.png)
Password is not readable but I guess it's to protect it from other eyes, right?

**Comment by chibat** [comment]
Are you using a web browser extension to change dark mode?

I look like this:
![image](https://user-images.githubusercontent.com/3880201/118401984-cd48f580-b6a2-11eb-9c84-60bb0e21b40a.png)

**Comment by salemalem** [comment]
No, I am not using any extension that makes it dark. I am using Brave browser 1.24.85 version for Windows 10.

**Comment by salemalem** [comment]
Maybe css error since CSS standards are different across browsers?
